### PR TITLE
[UPLOAD & FIXES] - Adding new views to the models and fixing some methods according to pep8

### PR DESCRIPTION
**Pull request by dacorredor11**

### CODE REVIEW

File: api/v1/views/index.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    """
+    Method that returns a Json with an Ok status
+    """
+
+    return jsonify({'Status': 'OK'})
+
+
+@app_views.route('/stats')
+def stats():
+    """
```
Comment (kajoomba99) [inline api/v1/views/index.py:35]: Here is missing the documentation

Comment (stevenbrand99) [inline api/v1/views/index.py:35]: Here is missing the documentation

**Review by stevenbrand99 — COMMENTED** [review]
Generally is good documented but there is a method in index.py that has no documentation

**Review by stevenbrand99 — CHANGES_REQUESTED** [review]
Generally is good documented but there is a method in index.py that has no documentation

File: api/v1/views/index.py (hunk 1 above)
Comment (dacorredor11) [reply api/v1/views/index.py:35]: Ok! I will add the required documentation

Comment (stevenbrand99) [reply api/v1/views/index.py:35]: You are 🐱‍🏍 great, thanks you the speed you correct this

**Pull request merged**